src/mmw/js/src/data_catalog/models.js

```diff
@@ -390,6 +390,93 @@ var SearchForm = Backbone.Model.extend({
     }
 });
 
+var CuahsiValue = Backbone.Model.extend({
+    defaults: {
+        source_id: '',
+        source_code: '',
+        quality_control_level_code: '',
+        value: null,
+        datetime: '',
+        date_time_utc: '',
+        time_offset: '',
+    }
+});
+
+var CuahsiValues = Backbone.Collection.extend({
+    model: CuahsiValue,
+});
+
+var CuahsiVariable = Backbone.Model.extend({
+    url: '/bigcz/values',
+
+    defaults: {
+        id: '',
+        name: '',
+        units: '',
+        display_name: '',
+        wsdl: '',
+        site: '',
+        values: null, // CuahsiValues Collection
+        most_recent_value: null,
+    },
+
+    initialize: function() {
+        this.set('values', new CuahsiValues());
+    },
+
+    search: function(from_date, to_date) {
+        var params = {
+                catalog: 'cuahsi',
+                wsdl: this.get('wsdl'),
+                site: this.get('site'),
+                variable: this.get('id'),
+                from_date: from_date,
+                to_date: to_date,
+            };
+
+        return this.fetch({
+            data: params,
+            processData: true,
+        });
+    },
+
+    parse: function(response) {
+        var mrv = null;
+
+        if (response.values) {
+            var values = this.get('values');
+
+            values.reset(response.values);
+            mrv = response.values[response.values.length - 1].value
```

Comment:
Is `response.values` guaranteed to have at least 1 value?

Reply:
Also, this line is missing a semi-colon. When the build works again, you may see some bad lint output.